modules/api/sync/package/data_provider.py

```diff
@@ -1,34 +1,34 @@
 import requests
 import json
 
+# URL de l'API à appeler
 DATASET='643d5f985c230e2b786c5602'
+url = 'https://demo.data.gouv.fr/api/1/datasets/' + DATASET
 
 def get_data():
-    # URL de l'API à appeler
-    url = 'https://demo.data.gouv.fr/api/1/datasets/' + DATASET
-
-    # Faire une requête GET pour récupérer le fichier
+    # requête pour récupérer la liste des ressources disponibles
     response = requests.get(url)
 
-    # Vérifier si la requête a réussi
-    if response.status_code == 200:
-        #print (response.content)
-        data = json.loads(response.text)
-        print( len(data['resources']))
-        nbfile = len(data['resources'])
-        for x in range(nbfile):
-            print( data['resources'][x]['url'])
-            file=data['resources'][x]['url']
-            response = requests.get(file)
-            if response.status_code == 200:
-        # Enregistrer le fichier sur le disque dur
-                with open(data['resources'][x]['title'], 'wb') as f:
-                    f.write(response.content)
-                print('Le fichier a été téléchargé avec succès.')
-            else:
-        # Afficher un message d'erreur si la requête a échoué
-                print('La requête a échoué avec le code d\'erreur', response.status_code)
-    else:
-        # Afficher un message d'erreur si la requête a échoué
+    if response.status_code != 200:
         print('La requête a échoué avec le code d\'erreur', response.status_code)
```

Comment:
Si possible changer le `print()` en `logger.error(Failed to download blah blah blah. Status code: %s'', response.status_code)`